pkg/client/imagesignatures.go

```diff
@@ -0,0 +1,41 @@
+package client
```

Comment:
Even more, there's a desire upstream to deprecate and remove the manual clients.